### PR TITLE
ipad menu fix

### LINKED ISSUE #59: Menu doesn't work in iPad screen
iPad pro 9.7 screen size

### PR DESCRIPTION
**Pull request by yashwin**
fixes #59

### CODE REVIEW

**Comment by yashwin** [comment]
Monthly and Yearly button is not working which is the new bug, I will fix that asap

File: js/main.js [hunk 1]
```diff
@@ -644,6 +644,25 @@ function resourceLinkClickAction(){
...
+    if (IS_IPAD || IS_IPHONE) {
+
+        $('.menubar').on('click touchend', function() {
+            var link = $(this).attr('href');
+            window.open(link,'_self'); // open in the same window
+
+            return true; // set false to prevent anchor click
+        });
+        $('#yearly').on('click',function(){
+           alert( 't!')
```
Comment (subho007) [inline js/main.js]: @yashwin is this for debugging? will you be removing this or should it be there?

File: js/main.js [hunk 2]
```diff
@@ -644,6 +644,25 @@ function resourceLinkClickAction(){
 }
 
 $(document).ready(function(){
+
+    IS_IPAD = navigator.userAgent.match(/iPad/i) != null;
+    IS_IPHONE = (navigator.userAgent.match(/iPhone/i) != null) || (navigator.userAgent.match(/iPod/i) != null);
+
+    if (IS_IPAD || IS_IPHONE) {
+
+        $('.menubar').on('click touchend', function() {
```
Comment (subho007) [inline js/main.js:677]: Nice solution @yashwin 👍

Comment (dhilipsiva) [inline js/main.js:677]: Will merge this when this debugging is removed.

**Comment by yashwin** [comment]
@subho007 : We need to add the **menubar** class in the blog page also.

**Pull request merged**